a_Calculate_Centers/src/calculate_AK_poi.R

```diff
@@ -57,26 +55,54 @@ calculate_AK_poi <- function() {
     Permanent_Identifier = character(),
     poi_Longitude = numeric(),
     poi_Latitude = numeric(),
-    poi_dist = numeric()
+    poi_dist_m = numeric()
   )
   for (i in 1:length(wbd[[1]])) {
-    coord = wbd[i,] %>% st_coordinates()
-    x = coord[,1]
-    y = coord[,2]
-    poly_poi = poi(x,y, precision = 0.00001)
     poi_df  <- poi_df %>% add_row()
+    one_wbd <- wbd[i,]
```

Comment:
I think that spaces after commas are part of the {tidyverse} style guide

```suggestion
    one_wbd <- wbd[i, ]
```